### DISCUSSION TITLE
Welcome to TourGuide Discussions!

### DISCUSSION THREAD

**Discussion by cdrock** [cdrock/TourGuide#77] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fredg1** [comment]
Want to post/ask something/report an issue but don't have a github account and don't want to make one?

Contact us in-game instead:
cdrock (owner) => cdrock (#2912644)
fredg1 (contributor) => fredg1 (#3041087)
phulin (contributor) => worthawholebean (#1972588)
